main.cpp

```diff
@@ -52,106 +52,106 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        amplitude
+//  action 1:    the amplitude mutes
+amplitude.mute();
+//  action 2:    the amplitude opens
+amplitude.open();
+//  action 3:    the amplitude increases
+amplitude.increase();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        cutoffFreq
+//  action 1:    the cutoffFreq increases
+cutoffFreq.increase();
+//  action 2:    the cutoffFreq decreases
+cutoffFreq.decrease();
+//  action 3:    the cutoffFreq is set to certain value
+cutoffFreq.set();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        kid
+//  action 1:    kid screams
+kid.scream();
+//  action 2:    kid runs
+kid.run();
+//  action 3:    kid paints
+kid.paint();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        cow
+//  action 1:    cow eats grass
+cow.eatGrass();
+//  action 2:    cow walks
+cow.walk();
+//  action 3:    cow swings tail
+cow.swingTail();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        musician
+//  action 1:    musician plays guitar
+musician.playGuitar();
+//  action 2:    musician changes guitar strings
+musician.changeGuitarStrings();
+//  action 3:    musician tunes guitar strings
+musician.tuneGuitarStrings();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        audience
+//  action 1:    audience watch something
+audience.watch();
+//  action 2:    audience aplaud something
+audience.aplaud();
+//  action 3:    audience dislike something
+audience.dislike();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        string
+//  action 1:    string vibrates
+string.vibrate();
+//  action 2:    string rings
+string.ring();
+//  action 3:    string breaks
+string.break();

```

Comment:
Sorry I didn't get it. I think I followed the requirement strictly.

like for
1) write out 10 nouns in the space provided below.
    Choose nouns that do things.  
    avoid choosing nouns that have things done TO them.
    If your noun consists of multiple words, use camelCaseToNameIt.

I did it, and did used multiple words noun in my second exercise, like cutoffFreq, volumeKnob. 


and for
 come up with a new noun and 3 actions
 pick a noun that needs camelCase for the name
 pick actions that need camelCase for their name

You didn't say I must make all noun consist of multiple words. And where did you mention I have to pick all actions with camelCase name? I re-checked the gumroad video and the description you left on the project, I only find these requirements:

 2) for each of the 10 nouns, write out 3 actions it might perform, in plain English.
 3) write out how you'd call that action in pseudo code, in the space after the plain-english action
 4) If the action requires multiple words, use camelCaseToNameIt

It's an IF, not a requirement for every action name.
So I did it in couple of my exercises. 


And for
Do not create an 11th noun.

I strictly followed the guide and wrote out 10 exercises in the blank part you left with numbers like 1) 2), you can count it out there're 10 added. The 11th one you might refer to the one YOU left in the example, like arm.extend() , you left it, it's not me. Maybe you want me to delete it? Then why don't you make it clearer if it's required?

Sorry my English is not that good, but I have tried my best to follow every step and requirement you asked, hope I explained it well.